point_pattern.py

```diff
@@ -56,9 +57,15 @@ def find_largest_city(gj):
     population : int
                  The population of the largest city
     """
-    city = None
+    temp = gj['features']
+    city = ""
     max_population = 0
 
+    for i in temp:
+        if (i['properties']['pop_max'] > max_population):
+            max_population = i['properties']['pop_max']
+            city = i['properties']['name']
+
     return city, max_population
```

Comment:
I image that something along these lines, but more robust will be part of your twitter work?